fastmigrate/core.py

```diff
@@ -242,14 +238,14 @@ def execute_sql_script(db_path: Path, script_path: Path) -> bool:
         
     except sqlite3.Error as e:
         # SQL error occurred
-        console.print(f"[bold red]Error[/bold red] executing SQL script {script_path}:")
-        console.print(f"  {e}", style="red")
+        sys.stderr.write(f"Error executing SQL script {script_path}:\n")
+        sys.stderr.write(f"  {e}\n")
         return False
     
     except Exception as e:
         # Handle other errors (file not found, etc.)
-        console.print(f"[bold red]Error[/bold red] executing SQL script {script_path}:")
-        console.print(f"  {e}", style="red")
+        sys.stderr.write(f"Error executing SQL script {script_path}:\n")
```

Comment:
Using `warn` or `logging` might be better since it's more flexible to enable/disable? Or if you definitely want to just write to stderr, probably `from sys import stderr; print(..., file=stderr)` would generally be better.

Reply:
Resolved via 5edea47